weaviate/collection/classes/config.py

```diff
@@ -32,18 +60,62 @@ class DataType(str, Enum):
     PHONE_NUMBER = "phoneNumber"
 
 
-class VectorIndexType(str, Enum):
+class _VectorIndexType(str, Enum):
     HNSW = "hnsw"
 
 
 class Tokenization(str, Enum):
+    """The available tokenization methods for text properties in Weaviate.
```

Comment:
Propose `The available inverted index tokenization methods for text properties in Weaviate.` to disambiguate from vectorizer tokenization (which I think ML people tend to think of first).